.circleci/config.yml

```diff
@@ -571,6 +571,11 @@ jobs:
       - run:
           name: Save last git commit message to .COMMIT
           command: git log -1 > .COMMIT
+      - save_cache:
+          name: Saving .git cache to be used on the E2E tests
+          key: gitCache-{{ .Branch }}
```

Comment:
Isn't this a terrible cache key? If someone is working on a feature branch, they can continue to push more commits and you certainly don't want to cache the previous `.git`.

Reply:
This was just a POC to open the thread we discussed in slack.